mail_layout_force/readme/CONFIGURE.rst

```diff
@@ -1,4 +1,4 @@
-# Go to Configuration > Technical > Emails > Templates
+# Go to Settings > Technical > Emails > Templates
 # Open the desired ``email.template`` record.
 # In Advanced Parameters tab, find the Force Layout field.
```

Comment:
```suggestion
#. Go to Settings > Technical > Emails > Templates
#. Open the desired ``email.template`` record.
#. In Advanced Parameters tab, find the Force Layout field.
```
The rendered layout is currently broken. Please apply the same to the list at the bottom.